rclpy/docs/source/api.rst

```diff
@@ -7,6 +7,7 @@ API
 
    api/init_shutdown
    api/node
+   api/duration
```

Comment:
It seems that you forgot to add the file `rclpy/docs/source/api/duration.rst`?

Reply:
Oops, that was an accidental commit. I was experimenting with building the docs to see what the generated documentation looks like. But since the result was rather useless, and I'm not familiar enough with Sphinx configuration to fix that, I did not plan to include it. 

Reverted.